build/lisp.tcl

```diff
@@ -1005,6 +1008,33 @@ respond "_" "lisp;_nilcom;defset\r"
 respond "_" "\032"
 type ":kill\r"
 
+# compile some lisp; libraries
+respond "*" "complr\013"
+respond "_" "lisp;_nilcom;cnvd\r"
+respond "_" "lisp;_lspsrc;exthuk\r"
+respond "_" "lisp;_lspsrc;gfile\r"
+respond "_" "lisp;_lspsrc;gfn\r"
+respond "_" "\032"
+type ":kill\r"
+
+respond "*" "complr\013"
+respond "_" "lisp;_lspsrc;querio\r"
+respond "_" "lisp;_lspsrc;vector\r"
+respond "_" "\032"
+type ":kill\r"
+
+respond "*" ":midas lisp;_lspsrc;sendi\r"
+expect ":KILL"
+respond "*" ":midas lisp;_lspsrc;straux\r"
+expect ":KILL"
+
+# lispt source is in libdoc, therefore fasl should be in liblsp
+# version in lisp; should be a link to liblsp;lispf fasl
+respond "*" ":delete lisp;lispt fasl\r"
+respond "*" ":link lisp;lispt fasl,liblsp;\r"
+respond "*" ":link sys2;ts lispt,sys2;ts edit\r"
+respond "*" ":link sys2;ts edit,sys2;ts emacs\r"
```

Comment:
The build fails due to SYS2; TS EDIT already existing.  It's created in emacs.tcl.